app/assets/javascripts/users.js.erb

```diff
@@ -158,6 +212,8 @@ function createTargetEvents() {
   $('#selected_area').change(changeTargetRadius);
 
   $('.principal-button').click(postNewTarget);
+
+  $('.home-container').on('click', '.destroy-target', destroyTarget)
```

Comment:
Why `$('.home-container').on('click', '.destroy-target', destroyTarget)` is inside `createTargetEvents()`?
You should put only logic related with the target creation on `createTargetEvents()`. Maybe you put this sentence in another function with a better name.